### PR TITLE
Fix pipeline issues

### PR DESCRIPTION
**Pull request by satyachundur**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/S28-882

### Change description ###
remove screeshots in build pipeline which is causing issue

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: pipeline/templates/run-playwright-test.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      arguments: '--filter ${{ parameters.testFilter }} --logger "console;verbosity=detailed"'
+      testRunTitle: '${{ parameters.testRunTitle }}'
+      projects: 'D:\a\1\s\test\e2e\pre.csproj'
+
+  #removing screenshots as its not needed because of videos exists
+  - task: CmdLine@2
+    continueOnError: true
+    timeoutInMinutes: 1
+    inputs:
+      script: 'dir D:\a\1\s\test\e2e\videos'
```
Comment (darrenbayliss-dev) [inline pipeline/templates/run-playwright-test.yml:16]: Part of these paths can be replaced with predefined variables - see here https://learn.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml

**Review by darrenbayliss-dev — COMMENTED** [review]
Just a few comments apart from that looks good.

File: pipeline/templates/run-playwright-test.yml (hunk 1 above)
Comment (satyachundur) [reply pipeline/templates/run-playwright-test.yml]: Added the build params

File: powerplatform-playwright-tests-on-test-par.yml [hunk 2]
```diff
@@ -1,41 +1,24 @@
...
-        vmImage: windows-latest 
+        vmImage: windows-latest
    jobs:
     - job: Test1_AdminManageCase
       timeoutInMinutes: 120
-
       steps:
       - template: include-build-playwright.yml
-    
       - task: CmdLine@2
```
[the remark this answers is not in the document]
Comment (satyachundur) [reply powerplatform-playwright-tests-on-test-par.yml]: Removed the time zone step

**Review by darrenbayliss-dev — APPROVED** [review]

**Pull request merged**